test/controllers/PaymentPlanSetUpControllerSpec.scala

```diff
@@ -42,7 +42,7 @@ class PaymentPlanSetUpControllerSpec extends ItSpec {
 
   List(
     Origins.Epaye.Bta,
-    Origins.Vat.Bta,
+    Origins.Vat.Bta
```

Comment:
does this need SA and SIA adding here?

Reply:
sa has it's own section, and currently sia doesn't really have any data to check

Reply:
if the tests for SA are doing the same thing as the ones for Epaye and Vat here then it might be worth exploring just adding in the tax regime to this list. What happens if you add those regimes to the list? Does it break?

Reply:
it breaks; I believe because sa has something different to the other two

Reply:
ah ok - is it possible to write SIA tests or do we need something else in place first?

Reply:
for now there wasnt even an siapaymentplan page; I have realised there probably should be something there, so I have made one real quick. for now it is basically identical to the sa page (until we know what it should be like).  adding a test in just so it's not forgotten when we get to changing the page